enhancements/installer/openstack-availibility-zones.md

```diff
@@ -0,0 +1,125 @@
+---
+title: openstack-availibility-zones
+authors:
+  - "@egarcia"
+  - "@pprenetti"
+reviewers:
+  - "@mandre"
+  - "@fedosin"
+approvers:
+creation-date: 2020-07-15
+last-updated: 2020-12-15
+status: implemented
+---
+
+# OpenStack Availability Zones
+
+## Release Signoff Checklist
+
+- [x]  Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [x] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+- Will the node's AZ be used for its volume? Will this effect the behavior of the cluster?
+
+## Summary
+
+The installer should automatically discover all availability zones (AZs) and distribute the control plane and compute nodes across them to maximize availibility by default. To accomodate use cases with AZ restrictions, the installer should also allow users to specify a set of them for each machinepool.
```

Comment:
While technically true, I find this part a bit confusing.

"machines spread across available AZs" is Nova's default, because the Installer does not pass any scheduler hint on machine creation (Nova distributes its instances anyway when [`default_schedule_zone`](https://docs.openstack.org/nova/train/configuration/config.html#DEFAULT.default_schedule_zone) is left to the default `None` in Nova).

I think we should look at this enhancement from a different perspective: what it does, is to give control to the user over instance placement. **With this enhancement, the user can control what Nova Availability zones the OpenShift servers are created on, and are bound to for their lifetime.**